packages/esm-patient-chart-app/src/banner/patient-banner.component.tsx

```diff
@@ -120,11 +121,10 @@ const PatientBanner: React.FC<PatientBannerProps> = ({
               {patient?.identifier?.length
                 ? patient?.identifier.map(({ value, type }) => (
                     <span className={styles.identifierTag}>
-                      <Tag key={value} type="gray" title={type.text}>
+                      <Tag className={styles.tag} key={value} type="gray" title={type.text}>
```

Comment:
I think it's usually best to keep `key` first in the list of attributes. In my mind it's conceptually different from (most of) the other attributes, since it's really there for React and not, properly speaking, a property of the rendered component (similar to, e.g., `ref`).